mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -19,9 +27,55 @@ import Utils from '../../common/utils/Utils';
 import ParallelCoordinatesPlotPanel from './ParallelCoordinatesPlotPanel';
 import { PageHeader } from '../../shared/building_blocks/PageHeader';
 import { CollapsibleSection } from '../../common/components/CollapsibleSection';
+import { shouldUseNextRunsComparisonUI } from '../../common/utils/FeatureUtils';
+import { useExperimentPageFeedbackUrl } from './experiment-page/hooks/useExperimentPageFeedbackUrl';
 
 const { TabPane } = Tabs;
 
+const LegacyCompareRunsPageCallout = (props) => {
+  LegacyCompareRunsPageCallout.propTypes = {
+    experimentIds: PropTypes.arrayOf(PropTypes.string).isRequired,
+  };
+  const feedbackFormUrl = useExperimentPageFeedbackUrl();
+  return shouldUseNextRunsComparisonUI() ? (
+    <Alert
+      type='info'
+      closable={false}
+      message={
+        <>
+          <FormattedMessage
+            defaultMessage='We’ve made several improvements to the new runs comparison experience.'
+            // eslint-disable-next-line max-len
+            description='Callout message to tell user about new compare runs feature'
+          />{' '}
+          <Link to={Routes.getExperimentPageRoute(props.experimentIds[0], true)}>
```

Comment:
Update "Give it a try" link to point to the chart view, rather than the table view